### DISCUSSION TITLE
E.T.A Longhorn’s V2 Data Engine to be production ready

### DISCUSSION THREAD

**Discussion by boedy** [longhorn/longhorn#7209] [Q&A]
I've been following the development of the V2 Data Engine and I'm quite impressed with what I've seen so far. I was wondering if there's any update on when it might be production-ready? I couldn't find this info anywhere and it would really help for some planning I'm doing.

Any rough timelines or updates you could share would be greatly appreciated. Thanks for your hard work on this project, it's really exciting to see it evolve!

**Comment by pcornelissen** [comment]
That would be interesting indeed (writing a comment to bump the bug)
